src/main/java/wooteco/subway/dao/jdbc/JdbcStationDao.java

```diff
@@ -26,14 +29,14 @@ public JdbcStationDao(JdbcTemplate jdbcTemplate) {
 
     @Override
     public Station create(Station station) {
-        String sql = "INSERT INTO station (name) VALUES(?)";
-        KeyHolder keyHolder = new GeneratedKeyHolder();
-        jdbcTemplate.update(connection -> {
-            PreparedStatement statement = connection.prepareStatement(sql, new String[]{"id"});
-            statement.setString(1, station.getName());
-            return statement;
-        }, keyHolder);
-        return new Station(keyHolder.getKey().longValue(), station.getName());
+        final SimpleJdbcInsert simpleJdbcInsert = new SimpleJdbcInsert(jdbcTemplate)
+                .withTableName("station").usingGeneratedKeyColumns("id");
```

Comment:
```suggestion
        final SimpleJdbcInsert simpleJdbcInsert = new SimpleJdbcInsert(jdbcTemplate)
                .withTableName("station")
                .usingGeneratedKeyColumns("id");
```
한줄에 점 하나씩 적는게 가독성이 더 높아질거 같아요!